ci/src/discord.py

```diff
@@ -39,5 +39,28 @@ def update_hook(webhook_url: str, info: dict, latest_ver: str, release: dict) ->
         embed['embeds'][0]["author"]["icon_url"] = f"{github_url}/{github_username}.png?size=40"
     release_notes = release.get('body')
     if release_notes and release_notes.strip():
-        embed['embeds'][0]['fields'].append({"name": "Release Notes", "value": release.get('body', "")})
-    requests.post(webhook_url, json=embed)
\ No newline at end of file
+        embed['embeds'][0]['fields'].append({"name": "Release Notes", "value": truncate_release_notes(release.get('body', ""), 1024)})
+    requests.post(webhook_url, json=embed)
+    
+def truncate_release_notes(release_notes: str, length: int) -> str:
```

Comment:
```suggestion
def truncate_release_notes(release_notes: str, length: int = MAX_BODY_LEN) -> str:
```
We should define "1024" as a constant so we don't use magic numbers. :)

Reply:
Yup good catch 😅, I think I let it slip cause I figured the surrounding logic was pretty tightly-coupled to the API anyway—will fix though

Reply:
No problem. This doesn't affect the code as much but helps readability later.